conans/cli/commands/build.py

```diff
@@ -34,12 +34,12 @@ def build(conan_api, parser, *args):
     deps_graph, lockfile = graph_compute(args, conan_api, partial=args.lockfile_partial)
 
     out = ConanOutput()
-    out.highlight("\n-------- Installing packages ----------")
+    out.highlight("-------- Installing packages ----------")
```

Comment:
I loved this extra blank line to separate the most important stages

Reply:
That is a good point, I definitely like it. Or some other way to make it really highlight (colors not enough), specially for reading, I find extremely useful to read the output.

Reply:
I created an `out.title` for the "----- XXXX ----" messages, all they will go with `\n` and automatic `----`.
I kept the `hightlight` for the `pink` messages.